zigpy/zcl/clusters/closures.py

```diff
@@ -434,11 +434,12 @@ class WindowCovering(Cluster):
     attributes = {
         # Window Covering Information
         0x0000: ("window_covering_type", t.enum8),
-        0x0001: ("physical_close_limit_lift_cm", t.uint16_t),
-        0x0002: ("physical_close_limit_tilt_ddegree", t.uint16_t),
+        0x0001: ("physical_closed_limit_lift_cm", t.uint16_t),
+        0x0002: ("physical_closed_limit_tilt_ddegree", t.uint16_t),
```

Comment:
Why "ddegree" here and in a few other attributes next to it?

Reply:
It was already there, and there is also "_cm".
"_cm" is for centimeters and I figured that "ddegree" is for decimal degrees as opposed to radians.

The ZCL spec does not add "Cm" nor "DDegree" in the attribute label.

Personally I would remove it as it's not in the original label, but as it was already there I left it in place.

If you all agree, I can remove those.

Reply:
They're called `PhysicalClosedLimit – Lift` and `PhysicalClosedLimit – Tilt` in the spec. The two subsequent attributes are named similarly.  I think `physical_closed_limit_lift` is fine, we don't provide units on any other attributes.

Reply:
I removed the _cm and _ddegree post fixes on the name.  I agree that the base names are sufficiently different.